RELEASE/scripts/autoscend/auto_quest_level_13.ash

```diff
@@ -205,88 +205,30 @@ boolean LX_getStarKey()
 	return autoAdv(1, $location[The Hole In The Sky]);
 }
 
-boolean L13_powerLevel()
```

Comment:
Why is `L13_powerLevel()` a bad name for a function that powerlevels to 13?

Reply:
It is not a bad name for a function that does that. But it is now that function does now. was only halfway doing it before too.

Our powerlevel code is split between two functions L13_powerLevel() and LX_attemptPowerLevel()
Some code is duplicated across both, some is unique to each. Particularly troublesome is the property tracking being split between the two.

Since I consolidated all the powerleveling code into LX_attemptPowerLevel()
This has left L13_powerLevel() without any power leveling code... well aside from calling the now sole powerleveling function LX_attemptPowerLevel()
Since it no longer contains powerleveling code I went ahead to rename it based on what it does now, which is start the final quest.

I don't mind changing the name back if you feel that name change is bad.

Reply:
I added that comment when I moved these functions out of autoscend.ash.
I apparently then renamed it from `L13_towerNSEntrance()` to the current `L13_powerLevel()` (which I have no recollection of doing).